src/app.js

```diff
@@ -60,6 +60,12 @@ export async function build() {
     }
   })
 
+  app.get('/refresh-seeds', async (req, res, next) => {
+    await parseTenantSeeds()
```

Comment:
If this is going to be named "parseTenantSeeds", fetching from URL should happen prior, and data should be passed to parseTenantSeeds. 

It could be renamed fetchAndUpdateTenantsFromUrl like this more like.